providers/src/airflow/providers/google/cloud/hooks/apache_kafka.py

```diff
@@ -0,0 +1,289 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""This module contains a Managed Service for Apache Kafka hook."""
+
+from __future__ import annotations
+
+from collections.abc import Sequence
+from copy import deepcopy
+from typing import TYPE_CHECKING
+
+from google.api_core.gapic_v1.method import DEFAULT, _MethodDefault
```

Comment:
I don't think we should import private functions from upstream libraries.

Reply:
@eladkal it's not a function it's a type class for `DEFAULT` value.  We do it for type checking for `retry` parameter and we use this `import` for each google's hook and operator. It's needed for the `DEFAULT` value which we also import by this line. The `DEFAULT` has type `_MethodDefault`. And we use this class `_MethodDefault` only for type checking.